app/graphql/types/record_type.rb

```diff
@@ -1,18 +1,18 @@
 module Types
   class LinkType < Types::BaseObject
-    field :kind, String, null: true
-    field :text, String, null: true
-    field :url, String, null: false
-    field :restrictions, String, null: true
+    field :kind, String, null: true, description: 'Type of link'
+    field :text, String, null: true, description: 'Additional description of a link, if applicable'
+    field :url, String, null: false, description: 'URL of a link'
+    field :restrictions, String, null: true, description: 'Restrictions on a link, if applicable'
```

Comment:
For this one, I think I'd prefer `if available`. We likely have restrictions on lots of links but it isn't in the metadata.